.github/workflows/build-pr.yaml

```diff
@@ -2,34 +2,30 @@ name: Build PRs
 on:
   pull_request:
 jobs:
-  build-all:
+  run-scripted-tests:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         java: [ '8', '11', '13' ]
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
+    
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    
     - name: Setup java
       uses: joschi/setup-jdk@v2
       with:
         java-version: ${{ matrix.java }}
         architecture: x64
+    
     - name: Cache SBT and Coursier cache
       uses: coursier/cache-action@v3
-    - name: Scalafmt check
-      run: |
-        cd core
-        sbt "scalafmtCheckAll; scalafmtSbtCheck"
-    - name: Build documentation
-      run: |
-        sudo snap install yq
-        cd docs
-        make html-author-mode
-    - name: Run integration tests
+    
+    - name: Run scripted tests
       run: |
         git config --global user.email "cloudflow@lightbend.com"
         git config --global user.name "Cloudflow CI"
         cd core
-        sbt -mem 2048 +test
```

Comment:
`core/tests` runs for `push.yaml` too.

Reply:
I agree on removing the duplicated `scalafmt` and `docs` build, but the `+test` is intentional and guarantees compatibility on the different JVMs

Reply:
Should we make the change there instead? I mean, make this run for multiple JVMs:

https://github.com/lightbend/cloudflow/blob/e9a48e29630df7f9629108539ed996d14eb89184/.github/workflows/push.yml#L26-L45

Reply:
I have no strong opinions here, I think I'll prefer it here, but I won't disagree on moving it to `push.yml`